backend/curriculum_tracking/tests/test_activity_log_entry_creators.py

```diff
@@ -40,3 +52,17 @@ def test_that_only_one_entry_created_for_one_review(self):
         creators.log_topic_competence_review_done(review2)
         creators.log_topic_competence_review_done(review2)
         self.assertEqual(LogEntry.objects.count(), 2)
+
+    def test_two_entries_created_for_two_reviews_with_second_done_within_debounce_period_by_another_user(self):
+        review1 = TopicReviewFactory()
+        review2 = TopicReviewFactory(
+            status=review1.status,
+            timestamp=review1.timestamp,
+            comments=review1.comments,
+            topic_progress=review1.topic_progress,
+            reviewer_user=UserFactory()
+        )
+
+        creators.log_topic_competence_review_done(review1)
+        creators.log_topic_competence_review_done(review2)
+        self.assertEqual(LogEntry.objects.count(), 2)
```

Comment:
Also test that if we log the same thing twice, it only gets logged once.
```
 creators.log_topic_competence_review_done(review1)
 creators.log_topic_competence_review_done(review1)
self.assertEqual(LogEntry.objects.count(), 1)
```

And another missing test is making sure that the log creators get envoked at the right time. When a new review is created then the correct log creators should be called.

Reply:
I've added extra tests now, the one however that I added I am not sure if you wanted it.  It basically checks that for a project review the log_topic_competence_review_done function cannot be invoked and vice versa with a topic review.